web/src/view/map/Room.tsx

```diff
@@ -79,7 +80,8 @@ export function Room ({ event, user }: RoomProps) {
   React.useEffect(() => {
     switchTable(getApolloClient(), {
       eventTableId: mainEventTableId,
-      participantId: user.user.id
+      participantId: user.user.id,
+      participantName: user.user.name
```

Comment:
should not need to pass user name, since it can be retrieved by user id. We want minimize our payload. Even though ik this might save a search on the server side, we should never rely on the client to supply anything more than essential information, since username could've been manipulated on the client side to be different from user id.

Reply:
got it